lib/components/viewers/nearby/stop.tsx

```diff
@@ -70,6 +69,12 @@ const Stop = ({
       const sortedStopTimes = st.stoptimes.sort(
         (a: StopTime, b: StopTime) => fullTimestamp(a) - fullTimestamp(b)
       )
+      if (
+        stopData.nearbyRoutes &&
+        !stopData.nearbyRoutes.includes(st?.pattern?.route?.gtfsId)
+      ) {
+        return <></>
```

Comment:
Can you add a comment(s) to better explain what's going on with this nearbyRoutes thing. It took me a while to really understand what's going on here and its relation to the service week thing